### PR TITLE
docs: update hacking.md to reflect new commands

### PR DESCRIPTION
**Pull request by kikisdeliveryservice**
Update commands in hacking.md since we now only have 1 Dockerfile/mco image.
Also add hack/push-image.sh command to hacking.md

Related-to: #739
Related-to: #948
Related-to: #950

### CODE REVIEW

File: docs/HACKING.md [hunk 1]
```diff
@@ -229,48 +229,40 @@ In those situations, you will need to:
...
+REPO={docker.io/username} /hack/push-image.sh
+```
 
 Quay.io or any other public registry isn't strictly required - you can use a local
 registry as long as those images are pullable.
 
 ## Build a custom release payload
 
-Now that your have your custom component images, to build a custom release payload, run:
+Now that your have your custom images, to build a custom release payload, run:
```
Comment (sgreene570) [inline docs/HACKING.md]: nit:
s/images/image

Comment (kikisdeliveryservice) [reply docs/HACKING.md]: ah thanks!

Comment (kikisdeliveryservice) [reply docs/HACKING.md]: done.

**Comment by kikisdeliveryservice** [comment]
note: this is going to fail verify until the fix is merged.

**Pull request merged**